meta-mbl-distro/conf/distro/mbl-production.conf

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+##################################################################################
+# mbl-production.conf
+#    Based on the main mbl.conf distribution
+#    Sets some defaults production variables options via mbl-distro-production.inc
+##################################################################################
+
+require conf/distro/mbl.conf
+require conf/distro/include/mbl-distro-production.inc
+
+# We add "mbl-production" to the DISTRO_FEATURES to make it visible to all recipes
+# and classes when the MBL Production Reference Distro is being used
+DISTRO_FEATURES += "mbl-production"
```

Comment:
Like I wrote above, I'm not convinced we need this or should have it.

Reply:
`DISTRO_FEATURES` dropped.